src/BbsBlsSignature2020.ts

```diff
@@ -51,12 +51,11 @@ export class BbsBlsSignature2020 extends suites.LinkedDataProof {
       throw new TypeError('"verificationMethod" must be a URL string.');
     }
     super({
-      type:
-        "https://w3c-ccg.github.io/ldp-bbs2020/context/v1#BbsBlsSignature2020"
+      type: "sec:BbsBlsSignature2020"
```

Comment:
Can you try and see if this works with the full IRI instead? e.g `https://w3id.org/security#BbsBlsSignature2020`

Reply:
These full IRI's didn't end up working because of the way the string matching works. We'll address this during the #81 though.